Converters/CSi/Speckle.Converters.CSiShared/CSiWrappers.cs

```diff
@@ -69,11 +78,11 @@ public static ICSiWrapper Create(int objectType, string name) =>
     {
       1 => new CSiJointWrapper { Name = name },
       2 => new CSiFrameWrapper { Name = name },
-      3 => new CSiCableWrapper { Name = name },
-      4 => new CSiTendonWrapper { Name = name },
+      3 => new CSiCableWrapper { Name = name }, // TODO: CSiCableWrapper
+      4 => new CSiTendonWrapper { Name = name }, // TODO: CSiTendonWrapper
       5 => new CSiShellWrapper { Name = name },
-      6 => new CSiSolidWrapper { Name = name },
-      7 => new CSiLinkWrapper { Name = name },
+      6 => new CSiSolidWrapper { Name = name }, // TODO: CSiSolidWrapper
+      7 => new CSiLinkWrapper { Name = name }, // TODO: CSiLinkWrapper
```

Comment:
I am curious on this wrappers. Do they needed to detect different objects in converters?

Reply:
Yeah, a bit of a workaround, since the API doesn't give us the **instances**, we can only query the selection.

<img width="325" alt="image" src="https://github.com/user-attachments/assets/a97ab895-d987-41af-b2e5-206995dcf39a">

So here we don't get objects, but an `int` telling us what `objectType` it is and an `objectName` which is unique for that `objectType` only. So, in order to be able to `ResolveConverter` we need to wrap this info into an "object" that keeps Etabs / Csi `name` and `type` dependencies.